sirepo/package_data/static/js/raydata.js

```diff
@@ -716,6 +722,14 @@ SIREPO.app.directive('scanSelector', function() {
                 });
             };
 
+            $scope.showArrowDown = (column) => {
```

Comment:
These two function contain almost the exact same logic (except for the `!`). So, I think using one function (ex `showArrowUp()` and then passing in a true/false based on whether or not to show the up arrow. If it is false then the down arrow will show.

Reply:
eb41875ad13fdf2a9ff2f5e261bcf5432c048ace